patterns/roks/module/variables.tf

```diff
@@ -136,6 +136,7 @@ variable "workers_per_zone" {
   default     = 1
 }
 
+# tflint-ignore: terraform_unused_declarations
```

Comment:
why are we ignoring this? If it is not used, it should not be here, but why is it not used?

Reply:
fixed